ui/cap-react/src/components/welcome/Header.js

```diff
@@ -66,51 +68,77 @@ class WelcomePage extends React.Component {
             </div>
           </Box>
           <Box direction="row" align="center" responsive={false}>
-            <Menu
-              dropAlign={{ top: "bottom" }}
-              size="small"
-              direction="row"
-              align="center"
-              pad={{ horizontal: "medium" }}
-            >
+            <MediaQuery minWidth={1070}>
               {Object.keys(this.props.nav).map((key, index) => (
-                <Box
-                  align="center"
-                  key={index}
-                  onClick={() =>
-                    this.props.scrollToRef(this.props.nav[key].ref)
-                  }
-                  className="menuItem"
-                >
-                  <Label>{this.props.nav[key].title}</Label>
+                <Box key={index}>
+                  <MenuItem
+                    title={this.props.nav[key].title}
+                    className="menuItem"
+                    background="inherit"
+                    href="#"
+                    onClick={() =>
+                      this.props.scrollToRef(this.props.nav[key].ref)
+                    }
+                  />
                 </Box>
               ))}
-              <Anchor href={this.state.oauthLink}>
-                <Label>Log in</Label>
-              </Anchor>
-            </Menu>
-            {process.env.NODE_ENV === "development" ? (
-              <Menu
-                dropAlign={{ top: "bottom" }}
-                icon={<LogIcon />}
-                size="small"
-                closeOnClick={false}
-              >
-                <LoginForm
-                  usernameType="email"
-                  defaultValues={{ username: "info@inveniosoftware.org" }}
-                  onSubmit={this.onFormSubmit}
-                />
-                {this.props.authError ? (
-                  <Box
-                    colorIndex="critical"
-                    margin={{ top: "small" }}
-                    pad="small"
-                  >
-                    {this.props.authError}
+              <MenuItem
+                title="Log in"
+                background="inherit"
+                href={this.state.oauthLink}
+                className="menuItem"
+              />
+            </MediaQuery>
+            <MediaQuery maxWidth={1069}>
+              <Menu top={55} right={0}>
```

Comment:
we should change the icon to a burger or something similar

Reply:
fixed, passed Burger Menu icon and background